float64 ts
uint32 allMotorInited
uint32[] motorInited
uint32[] ecatSlaveCommandsMsg_callbacked
uint32[] ecatSlaveCommandsMsg_enabled
string[] ecatSlaveCmdType
string[] ecatSlaveCmdTypePre
uint32[] subFsm
uint32[] subFsmPP
string[] statusword
string[] controlword
uint32[] fault
string[] error_code